nova/conf/base.py

```diff
@@ -112,6 +112,20 @@
 For a couple of operations, e.g. scheduling decisions and special settings when
 spawning, we have to identify a big VM and handle them differently. Every VM
 having more or equal to this setting's amount of RAM is a big VM.
+"""),
+    cfg.IntOpt(
+        'largevm_mb',
+        default=230 * 1024,      # 230 GB
+        min=0,
+        help="""
+Instance memory usage identifying it as large VM
+
+For a couple of operations, e.g. special settings when
+spawning, we have to identify a large VM and handle them differently - even
+differently than big VMs. Every VM having more or equal to this setting's
+amount of RAM and less than bigvm_mb is a large VM.
```

Comment:
We should definitely clean this terminology up (not just here, in general) once we're on rocky... I can only imagine the confusion for someone who didn't spend the last few months with this.

Reply:
Let's name them apples (big), pears (large) and oranges (small). People will like that :D

Reply:
Imagine that someone usually buys oranges that bigger than apples and likes small pears... :D